### ISSUE TITLE
PORT_WEB_UI is still leaving port 8080 mapped

### ISSUE THREAD

**Issue by unimonkey1994** [localstack/localstack#2386]
# Type of request: This is a ...

[X] bug report
[ ] feature request

# Detailed description
When setting option PORT_WEB_UI to use another port beside the default "8080" there is still a mapping for port 8080 remaining.

## Expected behavior
To be able to change the default port 8080 use completely so that other applications can use it.

## Actual behavior
Port 8080 is still being used.

# Steps to reproduce

## Command used to start LocalStack
% export PORT_WEB_UI=8888
% localstack start
Starting local dev environment. CTRL-C to quit.
docker run -it -e DEFAULT_REGION="us-east-1" -e LOCALSTACK_HOSTNAME="localhost" -e TEST_AWS_ACCOUNT_ID="000000000000" -e PORT_WEB_UI="8888" -p 443:443 -p 4566:4566 --rm --privileged --name localstack_main -p 8080:8080 -p 4567-4617:4567-4617 -p 10000:10000 -p 8888-8889:8888-8889  -v "/private/var/folders/sk/rynk5zgd67j6xfcp1hldwz_h0000gn/T/localstack:/tmp/localstack" -v "/var/run/docker.sock:/var/run/docker.sock" -e DOCKER_HOST="unix:///var/run/docker.sock" -e HOST_TMP_FOLDER="/private/var/folders/sk/rynk5zgd67j6xfcp1hldwz_h0000gn/T/localstack" "localstack/localstack"
...

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)

...

**Comment by whummer** [comment]
Thanks for reporting @unimonkey1994 . Can you please run the following command and give it another try:
```
pip install --upgrade --no-cache localstack localstack-ext localstack-client
```
If the problem persists, can you please provide another log dump of the `docker run -it ...` command that gets generated with the latest version. Thanks

**Comment by unimonkey1994** [comment]
It looks like it grabbed the latest version but still shows the same issue:

```
Successfully installed localstack-0.11.1.1 localstack-client-0.24 localstack-ext-0.11.12
% export PORT_WEB_UI=8888
% localstack start
LocalStack version: 0.11.1.1
Starting local dev environment. CTRL-C to quit.
docker run -it -e TEST_AWS_ACCOUNT_ID="000000000000" -e DEFAULT_REGION="us-east-1" -e PORT_WEB_UI="8888" -e LOCALSTACK_HOSTNAME="localhost" -p 4566:4566 --rm --privileged --name localstack_main -p 8080:8080 -p 4567-4620:4567-4620 -p 10000:10000 -p 8888-8889:8888-8889  -v "/private/var/folders/sk/rynk5zgd67j6xfcp1hldwz_h0000gn/T/localstack:/tmp/localstack" -v "/var/run/docker.sock:/var/run/docker.sock" -e DOCKER_HOST="unix:///var/run/docker.sock" -e HOST_TMP_FOLDER="/private/var/folders/sk/rynk5zgd67j6xfcp1hldwz_h0000gn/T/localstack" "localstack/localstack"
```

**Comment by GarySze** [comment]
I found the same problem.  The following workaround works for me:

Open the localstack Python source file: *Lib\site-packages\localstack_client\config.py*.  

Change the line:

`'dashboard': '{proto}://{host}:8080',`

to

`'dashboard': '{proto}://{host}:9090',`

Restart the localstack, the port 9090 is assinged to web ui, and port 8080 should no longer be occupied.

**Comment by whummer** [comment]
Thanks for the update @unimonkey1994 @GarySze . We've been able to reproduce the issue based on the information you've provided - this should now be fixed in 4014831e4dfcd41b3ea81d267c48c121bee6d0f9 . Can you please try installing version `0.11.3` of the pip package:
```
pip install --upgrade --no-cache 'localstack>=0.11.3'
```

Please report here if the problem persists. Thanks
